R/segmentation.R

```diff
@@ -288,8 +288,9 @@ setMethod(
   signature = c(x = 'Segmentation', y = 'SpatialPolygons'),
   definition = function(x, y, returnList = FALSE, fn = NULL, ...) {
     return(over(
```

Comment:
Need to check for `sf`
```suggestion
  definition = function(x, y, returnList = FALSE, fn = NULL, ...) {
    check_installed(pkg = 'sf')
    return(over(
```